test/pkg/addressable/send.go

```diff
@@ -60,11 +64,16 @@ func SendN(t *testing.T, n int, addressable Addressable) []string {
 				id := uuid.New().String()
 				event.SetID(id)
 
+				for _, mutator := range mutators {
+					mutator(&event)
+				}
+
 				client.Namespace = addressable.Namespace
 
-				client.SendEventToAddressable(ctx, fmt.Sprintf("%s-%d", addressable.Name, i), addressable.Name, &addressable.TypeMeta, event)
+				name := names.SimpleNameGenerator.GenerateName(fmt.Sprintf("%s-%d", addressable.Name, i))
```

Comment:
I think you don't need to generate a name bringing in this dep because SendEventToAddressable takes care of it

Reply:
No, it doesn't.

Reply:
Uhm my bad :smile: